WPFHospitalEditor/DynamicGridControl.xaml.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace WPFHospitalEditor
+{
+    /// <summary>
+    /// Interaction logic for DynamicGridControl.xaml
+    /// </summary>
+    public partial class DynamicGridControl : UserControl
+    {
+        private String[] contentRows;
+        private String separator;
+        private List<Label> labels = new List<Label>();
+        private List<TextBox> textBoxes = new List<TextBox>();
+        private Role role;
+        private Boolean isEquipmentOrMedicine;
+
+        public DynamicGridControl(String[] contentRows, String separator, Role role, Boolean isEquipmentOrMedicine)
+        {
+            InitializeComponent();
```

Comment:
I don't understand why you send separator in constructor if separator is always "=". You can define it as initial field value.

Reply:
I want to make it more flexible if separator is going to change in any time because this Dynamic Grid Control can be used in many situations if a view like that is needed.